near/src/runtime.rs

```diff
@@ -286,9 +295,12 @@ impl NightshadeRuntime {
             }
         };
 
+        let epoch_height = self.get_epoch_height_from_prev_block(prev_block_hash)?;
```

Comment:
The epoch height computed from the previous block might not be the same as the epoch height of the current block.

Reply:
epoch height from prev block meaning that given prev block hash we look up epoch id for next block. That's why it's not `get_epoch_height(prev_block_hash)` but `get_epoch_heiight_from_prev_block`. This is how epoch manager normally operated (see other get_..._from_prev_block()` functions.